kingpin/actors/aws/ecs.py

```diff
@@ -615,7 +668,7 @@ class Service(ECSBaseActor):
     """
 
     all_options = {
-        'state': (STATE, 'present', 'Desired state: present/absent'),
+        'desired_state': (STATE, 'present', 'Desired state: present/absent'),
```

Comment:
Okay, fair argument, but that's disappointing to me, since state as an option doesn't read like a verb to me. Something I can get used to though.